src/components/RichTextEditor/Plugins/MentionPlugin/MentionInputMarkupElement/MentionInputMarkupElementNode.tsx

```diff
@@ -0,0 +1,9 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { MentionInputElement, MentionInputElementProps, Value } from '@udecode/plate';
+import { MentionInputMarkupElementStyles } from './MentionInputMarkupElementStyles';
+
+export const MentionInputMarkupElementNode = (props: MentionInputElementProps<Value>) => (
```

Comment:
Don't forget `: ReactElement` so the return is ensured to the JSX and not something else
```suggestion
export const MentionInputMarkupElementNode = (props: MentionInputElementProps<Value>): ReactElement => (
```

Reply:
Updated